handlers.py

```diff
@@ -1,77 +1,61 @@
-import os
-import kopf
-import kubernetes
-import requests
-import constants
 from math import ceil
 from collections import namedtuple
+import requests
```

Comment:
you can try httpx and see if any side effect

Reply:
Yes, actually, it's the old code dependency that I wrote for the prototype. I'll remove it in the next PR. It is being used to poll flower API to fetch queue length. Autoscaling implementation will change entirely.